src/discussions/common/AuthorLabel.jsx

```diff
@@ -85,7 +85,7 @@ function AuthorLabel({
           />
           {authorLabelMessage && (
             <span
-              className={classNames('mr-1.5 font-size-14 font-style font-weight-500', {
+              className={classNames('font-size-14 font-style font-weight-500', {
```

Comment:
Please review if removing mr-1.5 has side effects at other places where it's been used